app/src/main/java/de/upb/hip/mobile/activities/ExhibitDetailsActivity.java

```diff
@@ -573,19 +577,19 @@ private void pauseAudioPlayback() {
         // TODO: integrate media player
         try {
             mMediaPlayerService.pauseSound();
-        } catch(IllegalStateException e){
-        } catch(NullPointerException e){
-        } catch(Exception e){
+        } catch (IllegalStateException e) {
+        } catch (NullPointerException e) {
+        } catch (Exception e) {
         }
         isAudioPlaying = false;
     }
 
-    private void stopAudioPlayback(){
-        try{
+    private void stopAudioPlayback() {
+        try {
             mMediaPlayerService.stopSound();
-        }catch(IllegalStateException e){
-        } catch(NullPointerException e){
-        } catch(Exception e) {
+        } catch (IllegalStateException e) {
```

Comment:
I know this is probably not your code, but we should never have exception handling like this:

1. Catches exceptions without handling them in any way
2. Catches the Generic "Exception e"